src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -940,6 +948,15 @@ bool FancyZones::IsNewWorkArea(GUID virtualDesktopId, HMONITOR monitor) noexcept
     return true;
 }
 
+bool FancyZones::IsSplashScreen(HWND window)
+{
+    TCHAR splashString[] = L"MsoSplash";
+    TCHAR className[MAX_PATH];
+    GetClassName(window, className, _countof(className));
+
+    return wcscmp(splashString, className) == 0;

```

Comment:
we need to check `GetClassName` return code, because if it's failed, we're comparing `splashString` to garbage in `className`.